### PR TITLE
Add tag support on (commit ,repo) combo

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java [hunk 1]
```diff
@@ -86,21 +87,33 @@ public int hashCode() {
...
-        + ModelDBUtils.getLocationWithSlashOperator(identifier.getLocationList());
+  public static String getVersioningCompositeId(
+      VersioningCompositeIdentifier identifier, IDTypeEnum.IDType idType) {
+    if (idType.equals(IDTypeEnum.IDType.VERSIONING_REPO_COMMIT_BLOB)) {
+      return identifier.getRepoId()
+          + "::"
+          + identifier.getCommitHash()
+          + "::"
+          + ModelDBUtils.getLocationWithSlashOperator(identifier.getLocationList());
+    } else {
```
Comment (ravishetye) [inline backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java]: else if

Comment (ravishetye) [reply backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java]: throw an exception in else

File: backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java [hunk 2]
```diff
@@ -86,21 +87,33 @@ public int hashCode() {
     }
   }
 
-  public static String getVersioningCompositeId(VersioningCompositeIdentifier identifier) {
-    return identifier.getRepoId()
-        + "::"
-        + identifier.getCommitHash()
-        + "::"
-        + ModelDBUtils.getLocationWithSlashOperator(identifier.getLocationList());
+  public static String getVersioningCompositeId(
```
Comment (ravishetye) [inline backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java]: getVersioningCompositeIdString

just differentiating the two function as Id and Identifier  is confusing

**Review by ravishetye — CHANGES_REQUESTED** [review]

File: backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java]: Fixed

File: backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java (hunk 2 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java]: Changed it to `getVersioningCompositeIdString` & `getVersioningCompositeId`

**Review by ravishetye — APPROVED** [review]

**Pull request merged**